### PR TITLE
update navi labels, add pre staking page

### PR DESCRIPTION
**Pull request by temptemp3**
## Summary by CodeRabbit

- **New Features**
	- Added a Deposit component to facilitate user deposits in the staking process.
	- Added a Withdraw component for users to withdraw funds from the staking contract.
	- Enhanced the Contract Picker to improve filtering and display of contracts based on specific criteria.
	- Introduced new constants for airdrop functionality.

- **Bug Fixes**
	- Streamlined the dashboard tab display by removing unnecessary routes and improving conditional rendering.

- **Style**
	- Enhanced the layout and design of the Deposit and Withdraw components for a better user experience.

- **Chores**
	- Removed outdated components and styles related to the lockup feature to simplify the codebase.

### CODE REVIEW

File: apps/web/src/App/Router/AppRouter.tsx [hunk 1]
```diff
@@ -92,11 +93,10 @@ function AppRouter(): ReactElement {
                           element={<Overview></Overview>}
                         ></Route>
                         <Route path="/stake" element={<Stake></Stake>}></Route>
-                        {/*
                         <Route
                           path="/deposit"
                           element={<Deposit></Deposit>}
-                    ></Route>*/}
+                        ></Route>
```
Comment (BenedictWHD) [inline apps/web/src/App/Router/AppRouter.tsx:99]: Aren't we removing the deposit view to have it as a component?

File: apps/web/src/App/Router/AppRouter.tsx [hunk 2]
```diff
@@ -114,10 +114,17 @@ function AppRouter(): ReactElement {
                           path="/airdrop"
                           element={<Airdrop></Airdrop>}
                         ></Route>
-                        {/*<Route
-                          path="/staking"
-                          element={<Staking></Staking>}
-                          ></Route>*/}
+                        {
+                          <Route
+                            path="/staking"
```
Comment (BenedictWHD) [inline apps/web/src/App/Router/AppRouter.tsx:119]: Node staking or Staking Program?

I think the paths should reflect the text on the main menu

Comment (temptemp3) [reply apps/web/src/App/Router/AppRouter.tsx:119]: staking-program

File: apps/web/src/App/Router/AppRouter.tsx (hunk 1 above)
Comment (temptemp3) [reply apps/web/src/App/Router/AppRouter.tsx:99]: Yeah I plan to remove these eventually

**Review by BenedictWHD — APPROVED** [review]

**Pull request merged**